.github/workflows/docs.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - uses: actions/setup-java@v2
         with:
           distribution: 'zulu'
-          java-version: 11
+          java-version: 17
```

Comment:
If we require Java 17 for compilation already, we should merge the change here quickly.

(if we still support compiling with Java 11, the main ci flow should ensure that, and we should merge anyway)

Reply:
It seems Java 17 is required to run docs flow now.
I had my https://github.com/trinodb/trino/pull/13209 failed, so will fix it there